components/collapsible-panel/README.md

```diff
@@ -75,6 +75,7 @@ The collapsible panel is a container that can show or hide additional content. I
 * Use to progressively disclose information or settings to simplify a complex workflow
 * Keep the header consistent across the collapsed and expanded states of the panel
 * Use the subtle collapsible panel for backgrounds that aren't white
+* Use [`d2l-collapsible-panel-group`](#d2l-collapsible-panel-group) to group multiple panels for better visual consistency
```

Comment:
@geurts has signed off on this point.

Don't think I did the link correctly....?

should it be `daylight.d2l.dev/components/collapsible-panel#d2l-collapsible-panel-group`?

Reply:
I'm not 100% sure, but I think it would need to be:
```
https://daylight.d2l.dev/components/collapsible-panel/#collapsible-panel-d2l-collapsible-panel
```

Reply:
Do you mean: 

```
https://daylight.d2l.dev/components/collapsible-panel/#collapsible-panel-group-d2l-collapsible-panel-group
```